src/foam/nanos/auth/token/TokenInvalidException.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  name: 'TokenInvalidException',
+  package: 'foam.nanos.auth.token',
+  extends: 'foam.nanos.auth.AuthenticationException',
+  javaGenerateDefaultConstructor: false,
+  javaGenerateConvenienceConstructor: false,
+
+  properties: [
+    {
+      name: 'exceptionMessage',
+      value: 'Token not found'
```

Comment:
Similar here.  
options are to rename to InvalidToken, so the english text becomes 
Invalid Token.